quant.module

```diff
@@ -520,40 +520,39 @@ function quant_seed($url, array $context = array()) {
 
   $queue = quant_get_queue();
 
+  // Pagination support.
   $document = new \DOMDocument();
   @$document->loadHTML($markup);
   $xpath = new \DOMXPath($document);
-  $page_urls = isset($context['page_urls']) ? $context['page_urls'] : [];
+  $xpath_urls = isset($context['xpath_urls']) ? $context['xpath_urls'] : [];
 
-  $pager_xpath = array(
-    '//a[contains(@href,"page=") and contains(text(), "next")]',
-    '//a[starts-with(@href, "/") and contains(text(), "first")]',
-  );
+  $xpath_selectors = array();
+  $links_config = variable_get('quant_xpath_selectors');
```

Comment:
For existing sites, we either need this to be defaulted or we need an update hook which I think would be better.

Reply:
Hi @kepol thanks for reviewing this! I have now updated variable_get function to use the default value which should resolve the issue for the existing sites.